test/common_test/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ set(TEST_TYPE "COMMON_TEST")
 set(tests
   basic_test
   construct_empty_world
+  simulation_features
 )
 
 link_directories(${PROJECT_BINARY_DIR}/test)
```

Comment:
Not related to this PR, but link_directories probably isn't necessary here?

Reply:
https://github.com/gazebosim/gz-physics/pull/362/commits/693a109fabbc8e73090ab71b9054dfebd443beef